packages/core/template/project/ManualTestingInstructions.md

```diff
@@ -0,0 +1,59 @@
+---
+  layout: default.md
+  title: "Manual Testing Instructions"
```

Comment:
I would delete this file from appendix as it seems to overlap with testing (or move it under project guides)